src/js/media-time-range.js

```diff
@@ -25,47 +25,49 @@ const template = document.createElement('template');
 template.innerHTML = `
   <style>
     #thumbnailContainer {
-      display: none;
+      position: absolute;
+      left: 0;
+      top: 0;
+      transition: visibility .25s, opacity .25s;
+      visibility: hidden;
+      opacity: 0;
     }
 
     media-thumbnail-preview {
-      /* Scale the thumbnail preview to 50% and reposition appropriately to not take up too much real-estate. */
-      transform: scale(0.5) translateY(50%) translateX(-100%);
+      --thumb-min-width: var(--media-thumbnail-preview-min-width, 120px);
+      --thumb-max-width: var(--media-thumbnail-preview-max-width, 200px);
+      --thumb-min-height: var(--media-thumbnail-preview-min-height, 80px);
+      --thumb-max-height: var(--media-thumbnail-preview-max-height, 160px);
+      transform-origin: 50% 100%;
       position: absolute;
       bottom: calc(100% + 5px);
       border: 2px solid #fff;
       border-radius: 2px;
       background-color: #000;
     }
 
-    /* Can't get this working. Trying a downward triangle. */
+    /*
+      This is a downward triangle. Commented out for now because it would also
+      require scaling the px properties below in JS; bottom and border-width.
+    */
     /* media-thumbnail-preview::after {
       content: "";
       display: block;
-      width: 300px;
-      height: 300px;
-      margin: 100px;
-      background-color: #ff0;
+      width: 0;
+      height: 0;
+      position: absolute;
+      left: 50%;
+      transform: translateX(-50%);
+      bottom: -10px;
+      border-left: 10px solid transparent;
+      border-right: 10px solid transparent;
```

Comment:
non-blocking: you can also use [`border-inline`](https://developer.mozilla.org/en-US/docs/Web/CSS/border-inline) to have this as one item.